lib/matplotlib/axes/_base.py

```diff
@@ -118,7 +118,7 @@ def __call__(self, ax, renderer):
             self._transform - ax.figure.transSubfigure)
 
 
-def _process_plot_format(fmt):
+def _process_plot_format(fmt, *, ambiguous_fmt_datakey=False):
```

Comment:
Bonus points if you copy the docstring for the parameter here as well.

Reply:
The docstring is already not numpydoc'ed (and doing properly so is slightly messy due to, well, the messy API), so I'll skip these points for now.